MAUI/ImageEditor/overview.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: About .NET MAUI ImageEditor control | Syncfusion

```

Comment:
Overview of .NET MAUI ImageEditor control | Syncfusion

Reply:
title changed